src/goto-programs/goto_convert_side_effect.cpp

```diff
@@ -445,7 +430,7 @@ void goto_convertt::remove_cpp_delete(
   side_effect_exprt &expr,
   goto_programt &dest)
 {
-  assert(expr.operands().size()==1);
+  DATA_INVARIANT(expr.operands().size() == 1, "cpp_delete expected 1 operand");
```

Comment:
Let's spell out "one", for consistency. Or change all of them to numerals. There's ample inconsistency in this PR as far as this is concerned.

Reply:
Agreed. I will handle all of the cases in this PR.